sdk/eventhub/event-hubs/test/activate-browser-logging.ts

```diff
@@ -0,0 +1,18 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { afterAll, beforeAll } from "vitest";
+import { AzureLogLevel, setLogLevel } from "@azure/logger";
+
+const logLevel = (process.env.AZURE_LOG_LEVEL as AzureLogLevel) || "info";
```

Comment:
nit: give it a type?

```suggestion
const localStorage: { debug?: string } = {};
```

Reply:
addressed in https://github.com/Azure/azure-sdk-for-js/pull/30421/commits/b2b76797cc8c919cd8084e934fd560c8ac2bff55